catalogue/webapp/components/WorksSearchResults/WorksSearchResults.js

```diff
@@ -26,7 +26,7 @@ const WorkSearchResults = ({ works, worksRouteProps, apiProps }: Props) => {
   const { relevanceRating } = useContext(TogglesContext);
 
   return (
-    <div className={'grid'} data-test-id="search-results">
+    <div className={'grid'} role="search results list">
```

Comment:
Search is the only valid role to use here (landmark role): https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques#Roles

results and list can't be used as role values

Reply:
although the list of results should be an html list so role of list is probably redundant

Reply:
Thanks for this .. makes sense based on the discussion we had.. will update the selectors and roles which makes sense.